.idea/MenuMaker.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think that .gitignore is not working

Reply:
It is, those are global configuration files for anyone else who is using WebStorm.